src/components/KPop/KPop.vue

```diff
@@ -28,7 +28,7 @@
           v-show="isVisible"
           :id="popoverId"
           ref="popoverElement"
-          :aria-labelledby="$slots.title || title ? titleId : undefined"
+          :aria-labelledby="slots.title || title ? titleId : undefined"
```

Comment:
Wondering if we can make this `computed` since there were 4 references of this condition.

Reply:
Noted, but since it’s a simple expression, I feel a computed is overkill and adds unnecessary overhead.